wait/member.go

```diff
@@ -163,6 +170,25 @@ func (a *MemberAwaitility) WaitForNamespace(username, typeName, revision string)
 	return &ns, nil
 }
 
+// WaitForRoleBinding waits until a RoleBinding with the given name exists in the given namespace
+func (a *MemberAwaitility) WaitForRoleBinding(namespace *v1.Namespace, name string) (*rbacv1.RoleBinding, error) {
+	roleBinding := &rbacv1.RoleBinding{}
+	err := wait.Poll(a.RetryInterval, a.Timeout, func() (done bool, err error) {
+		obj := &rbacv1.RoleBinding{}
+		if err := a.Client.Get(context.TODO(), types.NamespacedName{Namespace: namespace.Name, Name: name}, obj); err != nil {
+			if errors.IsNotFound(err) {
+				a.T.Logf("waiting for availability of RoleBinding '%s' in namespace '%s'", name, namespace.Name)
+				return false, nil
+			}
+			return false, err
+		}
```

Comment:
See https://github.com/codeready-toolchain/toolchain-e2e/commit/77c23fb4b10732473b141719e50b61cc4459d9fb#r36024055

Reply:
Done in https://github.com/codeready-toolchain/toolchain-e2e/pull/49/commits/fe617339bb2123574f4a6f4c86387c64c2efdde3
(added better logging with the list of current role binding when waiting for a role binding)

> There are many namespaces which not related to toolchain. So we want to filter them. But there should not be any RoleBindings which are not created by toolchain in the user namespaces. So we don’t want to filter them (in error logs). But listening all of them might be helpful, yes. I was just lazy to add it :)

There are actually other role bindings too: system:image-pullers, etc. So I added the provider filter.